setup.cfg

```diff
@@ -2,7 +2,6 @@
 name = cobalt
 
 [tool:pytest]
-python_paths=.
```

Comment:
Wouldn't this break the ability to just run `pytest` from the repo root ?

Reply:
Tried both unsetting PYTHONPATH and running with vscode test discovery, both worked